s2spy/rgdr/rgdr.py

```diff
@@ -343,7 +343,7 @@ def __init__(
             min_area_km2 (float): The minimum area of a cluster. Clusters smaller than
                 this minimum area will be discarded.
         """
-        self._clusters = None
+        self.cluster_map = None
```

Comment:
also initialize other new attrs as None.

Reply:
A small note @semvijverberg , if you have linters configured in your coding environment, then it will warn you that "Attribute 'corr_map' defined outside __init__pylint(attribute-defined-outside-init)". So it is nice to work with an editor with python linters/extensions configured 😜 .

Reply:
Done.